spec/rails_spec.rb

```diff
@@ -20,7 +20,7 @@ def sh(command, options={})
       end
 
       Dir.chdir("spec/fixtures/#{rails}") do
-        Bundler.with_clean_env do
+        Bundler.with_unbundled_env do
```

Comment:
alternatively:
```suggestion
        method = (Bundler.respond_to?(:with_unbundled_env) ? :with_unbundled_env : : with_clean_env)
        Bundler.send(method) do
```

Reply:
Since this is just a spec, if I follow your previous suggestion, I think it shouldn't be needed?

Reply:
yeah, should not be needed, just as easy-way-out if you are still stuck

Reply:
CI is green, so marking this as "resolved" :)